build_tools/third_party/swiftshader/build_vk_swiftshader.sh

```diff
@@ -54,7 +54,7 @@ else
   SWIFTSHADER_INSTALL_DIR="${BASE_DIR?}"'\.swiftshader'
 fi
 
-SWIFTSHADER_COMMIT=c27e99245d423648fe5f0eafa73d49ce7b53d201
+SWIFTSHADER_COMMIT=57eca9df5dc56d290f662ace7481ed758618d559
```

Comment:
This needs to be updated in https://github.com/google/iree/blob/main/build_tools/docker/swiftshader/Dockerfile too, and all the docker images rebuilt (sorry).

See the README at https://github.com/google/iree/tree/main/build_tools/docker/

Reply:
Images updated.